catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SolrCache.java

```diff
@@ -292,6 +294,16 @@ public void removeById(String[] ids) throws IOException, SolrServerException {
     return getMetacardsFromResponse(response);
   }
 
+  public void setAnyTextAttributes(List<String> anyTextAttributes) {
```

Comment:
❗️ We are clearing the content if the list to add items in it. What is the though about multi-threading here?